### ISSUE TITLE
WEBSITE FOLDER [& FILE] create : lighten nesting

### ISSUE THREAD

**Issue by Skaant** [Skaant/motifs-js#38]
Allow 'a/b/c' nesting to be created automatically to avoid :

```javascript
folderMotif.create(
  '_tests',
  'article-create-new',
  folderScope => [
    folderMotif.create(
      folderScope,
      '_data',
      folderScope => [
        folderMotif.create(
          folderScope,
          'articles',
          folderScope => [
            folderMotif.create(
              folderScope,
              '1',
              folderScope => [
                fileMotif.create(
                  folderScope,
                  '1.article.js',
                  () => ''
                )
              ]
            ) 
          ]
        )
      ]
    )
  ]
)
```

**Comment by Skaant** [comment]
FILE should have a safe-creation mode, **creating required folder from its path**

**Comment by Skaant** [comment]
Feature implemented in `0.4.0`. See new syntax below :
